bin/license.py

```diff
@@ -58,7 +58,13 @@
     "javax.servlet.jsp:jsp-api:2.1": "(CDDL + GPLv2 with classpath exception) Java Servlet API (javax.servlet.jsp:jsp-api:2.1 - https://javaee.github.io/javaee-jsp-api)",
     "javax.servlet:servlet-api:2.5": "(CDDL + GPLv2 with classpath exception) Java Servlet API (javax.servlet:servlet-api:2.5 - http://servlet-spec.java.net)",
     "oro:oro:2.0.8": "(Apache License, Version 1.1) ORO (oro:oro:2.0.8 - https://mvnrepository.com/artifact/oro/oro)",
-    "org.hyperic:sigar:1.6.5.132": "(Apache License, Version 2.0) Sigar (org.hyperic:sigar:1.6.5.132 - https://github.com/hyperic/sigar)"
+    "org.hyperic:sigar:1.6.5.132": "(Apache License, Version 2.0) Sigar (org.hyperic:sigar:1.6.5.132 - https://github.com/hyperic/sigar)",
+    "asm:asm:3.1": "(BSD License) ASM (asm:asm:3.1 - https://asm.ow2.io/license.html)",
+    "com.ibm.icu:icu4j:55.1": "(Unicode/ICU License) ICU4J (com.ibm.icu:icu4j:55.1 - http://icu-project.org/)",
+    "jakarta.activation:jakarta.activation-api:1.2.1": "(EDL 1.0) Jakarta Activation API (jakarta.activation:jakarta.activation-api:1.2.1 - https://github.com/eclipse-ee4j/jaf)",
+    "org.apache.zookeeper:zookeeper:3.3.1": "(Apache License, Version 2.0) Apache ZooKeeper - Server (org.apache.zookeeper:zookeeper:3.3.1 - http://zookeeper.apache.org/zookeeper)",
+    "org.apache.zookeeper:zookeeper:3.4.6": "(Apache License, Version 2.0) Apache ZooKeeper - Server (org.apache.zookeeper:zookeeper:3.4.6 - http://zookeeper.apache.org/zookeeper)",
+    "org.codehaus.jettison:jettison:1.1": "(Apache License, Version 2.0) Jettison (org.codehaus.jettison:jettison:1.1 - https://github.com/jettison-json/jettison)"
```

Comment:
@asdf2014 I would like to know, under what circumstances will these contents be added?

Reply:
Only needed when some licenses are ambiguous and need to be determined manually

Reply:
> Only needed when some licenses are ambiguous and need to be determined manually

ok, got it, thanks